File: non_constructible_change.py
def nonConstructibleChange(coins):
    # sort coins
    coins.sort()
    # initialize a smallest_change variable as coins[0]
    largest_change = 0
    # iterate through coins
    for coin in coins:
        if coin > largest_change + 1:
            return largest_change + 1
        largest_change += coin

    return largest_change + 1

File: easy/nonConstructibleChange.js
function nonConstructibleChange (coins) {
  coins.sort();
  let change = 0;
  for (let i = 0; i < coins.length; i ++) {
    if (coins[i] > change + 1) {
      return change + 1;
    } else {
      change += cooins[i];
    };
  };
  return change + 1;
};